src/components/Widgets/AlertProvider/index.jsx

```diff
@@ -0,0 +1,112 @@
+import React, {
```

Comment:
Please add appropriate license and copyright header.

Reply:
Sorry for messing up this pull request. I have made the requested changes in a new pull request kindly review it. #186